lib/WebSocketFrame.js

```diff
@@ -242,20 +242,20 @@ WebSocketFrame.prototype.toBuffer = function(nullMask) {
 
     if (this.length > 125 && this.length <= 0xFFFF) {
         // write 16-bit length
-        output.writeUInt16BE(this.length, outputPos, true);
+        output.writeUInt16BE(this.length, outputPos);
         outputPos += 2;
     }
     else if (this.length > 0xFFFF) {
         // write 64-bit length
-        output.writeUInt32BE(0x00000000, outputPos, true);
-        output.writeUInt32BE(this.length, outputPos + 4, true);
+        output.writeUInt32BE(0x00000000, outputPos);
+        output.writeUInt32BE(this.length, outputPos + 4);
         outputPos += 8;
     }
 
     if (this.mask) {
-        maskKey = nullMask ? 0 : (Math.random()*0xFFFFFFFF) | 0;
-        this.maskBytes.writeUInt32BE(maskKey, 0, true);
-        
+        maskKey = nullMask ? 0 : (Math.random()*0xFFFFFF) | 0;
```

Comment:
Why are you reducing the size of the mask key from 32-bits to 24-bits?

Reply:
This technically reduces the security of the websocket protocol, causing one byte out of every four to be unmasked. I'd like to approve this PR but this change can't be part of it.